File: mordisco-api/src/main/java/utn/back/mordiscoapi/model/dto/calificaciones/CalificacionRepartidorResponseDTO.java
package utn.back.mordiscoapi.model.dto.calificaciones;

import java.time.LocalDateTime;

public record CalificacionRepartidorResponseDTO(
        Long id,
        Long pedidoId,
        Long repartidorId,
        String repartidorNombre,
        Integer puntajeAtencion,
        Integer puntajeComunicacion,
        Integer puntajeProfesionalismo,
        Double puntajePromedio,
        String comentario,
        LocalDateTime fechaHora,
        String clienteNombre
) {}

File: mordisco-front/src/app/shared/models/calificacion/calificacion-repartidor-response-dto.ts
import { ICalificacionBase } from "./calificacion-base";

export default interface CalificacionRepartidorResponseDTO extends ICalificacionBase{
    repartidorId : number,
    repartidorNombre : string,
    puntajeAtencion : number,
    puntajeComunicacion : number,
    puntajeProfesionalismo : number,

}      
